### DISCUSSION TITLE
Green box widget for Android/IOS

### DISCUSSION THREAD

**Discussion by codad5** [community/community#80930] [Mobile]
It would be nice if android can have the GitHub activity green box widget, so that once can be able to add it to their holme screen .

An app [GitHub Widget](https://play.google.com/store/apps/details?id=jp.co.chlorocube.githubcontributions) perform this but it would br nice if in built

Thanks

**Comment by kangarruu** [comment]
@codad5, thanks for taking the time to send us your suggestions! We are continuously working on adding more features to our app to make your experience better. I'll add your request to our tracker so that we can prioritize this request in the future!

**Reply by kangarruu** [reply]
For clarification, we do support a contribution widget in GitHub Mobile for iOS (read more [here](https://github.blog/changelog/2021-02-02-github-for-ios-contribution-graph-widget/)) But I'll let the team know there's interest in adding this to our Android version as well!

**Reply by codad5** [reply]
Ok, thanks, would be hoping to see it anytime soon
